lib/go-tc/cachegroup_parameter_associations.go

```diff
@@ -0,0 +1,44 @@
+package tc
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// CacheGroupParametersResponse is a Cache Group Parameter response body.
+type AllCacheGroupParametersResponse struct {
+	Response CacheGroupParametersList `json:"response"`
+	Alerts
+}
+
+type CacheGroupParametersList struct {
+	CacheGroupParameters []CacheGroupParametersNullable `json:"cachegroupParameters"`
+}
+
+// CacheGroupParameter ...
+type CacheGroupParameters struct {
+	CacheGroup  int       `json:"cachegroupId"  db:"cachegroup"`
```

Comment:
FWIW I believe `cachegroupId` was handled as `cacheGroupId` in Perl we should match that

Reply:
Yeah, there was a typo in the docs example, which I tried to fix in time for someone rewriting this endpoint to see, but https://github.com/apache/trafficcontrol/pull/4329 still has to get merged....

Reply:
done